src/parser/shaman/enhancement/modules/talents/ForcefulWinds.js

```diff
@@ -5,13 +5,10 @@ import { formatNumber, formatPercentage } from 'common/format';
 
 import Analyzer from 'parser/core/Analyzer';
 
-import StatisticBox from 'interface/others/StatisticBox';
+import TalentStatisticBox from 'interface/others/TalentStatisticBox';
 import calculateEffectiveDamage from 'parser/core/calculateEffectiveDamage';
 
-
-const FORCEFUL_WINDS = {
-  INCREASE: 1,
-};
+const FORCEFUL_WINDS_INCREATE = 0.8;
```

Comment:
Spelling :) Last thing before it looks good to go (from my pov)

Reply:
Has been adjusted